roles/etcd_prometheus_exporter_cert/tasks/gen_cert.yml

```diff
@@ -0,0 +1,85 @@
+- name: 'make assertions'
+  assert:
+    that:
+     - groups.etcd is defined
+
+- name: 'create private key on master'
+  command: openssl genrsa -out {{ etcd_exporter_key }} 4096
+  args:
+    creates: '{{ etcd_exporter_key }}'
```

Comment:
Does this result in this being done only once? We don't want to re-generate the cert on every invocation of the playbook, right?